src/CharterManager.sol

```diff
@@ -282,10 +282,10 @@ contract CharterManagerImp {
 
     function onVatFlux(address gemJoin, address from, address to, uint256 wad) external {}
 
-    function quit(bytes32 ilk, address dst) external {
+    function quit(bytes32 ilk, address u, address dst) external allowed(u) {
```

Comment:
Similar to the `exit(...)` I'm not sure if we should break interface parity here. However, this is kind of a special exception as it's not identical to how this works with the cdp manager.

I think I'm okay if we decide to leave this.

Reply:
As you mentioned this is different than the `exit()` since we would have needed to `fork()` the position before `quit()`.
Also the cdp-manager does use similar delegation with quit, so I think it's fine to leave as is.

Reply:
We might consider as Sam pointed out, to actually implement a fully `fork` than just `quit`.
This last one would be probably built on the wrapper cdp manager.

Reply:
Remember that's only for when the vat is not live.
As in any other case we don't want to allow to fork to any address (that for example is not managed by charter manager).
So not sure that entitles/deserves implementing the fork.

Reply:
So also to summarize the discussion with @gbalabasquer about this -
The use of `quit` here is to move from a proxied `urn` to a non proxied one. This is needed only in ES.
We don't want to allow that in other conditions since we want to keep collateral and debt only in urns managed by charter.

A standalone/passthrough `fork` function here might be a one that just moves from one proxied `urn` to another. But we don't really see a usecase for that, and on the contrary it might allow to move from one fee program to another.
It will also not help for the quit scenario I mentioned.

We tend to leave this as is.